_rules/SVG-image-has-name

```diff
@@ -0,0 +1,145 @@
+---
+name: SVG image has name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any `SVG` element that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element has a non-empty accessible name.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+Browser and assitive technology support for SVG `title` and `desc` elements is currently inconsistent. Using WAI ARIA in combination with the `img` role for non-decorative `svg` elements significantly.
```

Comment:
Is there something missing after "significantly"? The last sentence seems to have been cut short.

Reply:
Yup sorted